src/components/FormContainer.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { loginSignupFunction } from "../actions/authorization";
+
+class FormContainer extends Component {
+  initialState = {
```

Comment:
It should be a static field otherwise it will eat memory with any instance of this class.
As I see you don't use initialState in any other way besides initialization of component.
It means you need just:

```
class FormContainer extends Component {
  state = {
    name: "",
    password: "",
  }
}
```